packages/rum/src/domain/segmentCollection/segmentCollection.ts

```diff
@@ -1,13 +1,21 @@
 import type { DeflateEncoder, HttpRequest, TimeoutId } from '@datadog/browser-core'
-import { isPageExitReason, ONE_SECOND, clearTimeout, setTimeout } from '@datadog/browser-core'
+import {
+  isPageExitReason,
+  ONE_SECOND,
+  clearTimeout,
+  setTimeout,
+  isExperimentalFeatureEnabled,
+  ExperimentalFeature,
+} from '@datadog/browser-core'
 import type { LifeCycle, ViewHistory, RumSessionManager, RumConfiguration } from '@datadog/browser-rum-core'
 import { LifeCycleEventType } from '@datadog/browser-rum-core'
 import type { BrowserRecord, CreationReason, SegmentContext } from '../../types'
 import { buildReplayPayload } from './buildReplayPayload'
 import type { FlushReason, Segment } from './segment'
 import { createSegment } from './segment'
 
-export const SEGMENT_DURATION_LIMIT = 30 * ONE_SECOND
+export const SEGMENT_DURATION_LIMIT = () =>
+  isExperimentalFeatureEnabled(ExperimentalFeature.REDUCE_SEGMENT_LIMIT_BATCH_TIME) ? 5 * ONE_SECOND : 30 * ONE_SECOND
```

Comment:
* Top level functions should be declared as `function ...(){}`, not arrow functions.
* Function names should be in camel case, only top level constants primitive values should be in upper case.

Let's find some eslint rule to enforce that. In the meantime:

**💬 suggestion:** ‏

```suggestion
export function getSegmentDurationLimit() {
  return isExperimentalFeatureEnabled(
    ExperimentalFeature.REDUCE_SEGMENT_LIMIT_BATCH_TIME) ? 5 * ONE_SECOND : 30 * ONE_SECOND
  )
}
```